src/pages/dashboard/[subdomain]/editor.tsx

```diff
@@ -148,7 +148,15 @@ export default function SubdomainEditor() {
         toast.error("Error: " + createOrUpdatePage.data?.message)
       }
     }
-  }, [createOrUpdatePage.isSuccess])
```

Comment:
I left out others on purpose, otherwise, the change in others would cause the toast to reappear

Reply:
Thx for the explication, i understand why you left them out, i thank if we use "useEffect" more than twice is a good idea.